newspack-theme/sass/blocks/_blocks.scss

```
@@ -1694,6 +1694,16 @@ hr {
 			}
 		}
 	}
+
+	// Make SCAIP placements full-width.
+	.entry .entry-content .scaip {
+		position: relative;
+		background: rgba( 0, 0, 0, 0.05 );
+		max-width: none;
+		width: 100vw;
+		margin-left: -50vw;
+		left: 50%;
```

These styles look like they get applied to all SCAIP-inserted widgets, not just ads - I know this used to be a think more publishers used prior to campaigns taking off, but we still might want to avoid non-ad widgets. In that case, something like this should work:

```suggestion
	.entry .entry-content .scaip .widget_newspack-ads-widget {
		background: $color__background-pre;
		margin-left: calc( 50% - 50vw );
		margin-right: calc( 50% - 50vw );
		max-width: 100vw;
	}
```

(I also made a couple tweaks for consistency -- using the same light grey Sass variable as in other spots, and the same styles to make the ad full width as we use for the full-width blocks!). 

This _will_ break the custom colours styles -- they'll also have to be updated to be more specific in the inc/color-patterns.php file.